specs/protocol/tx_format.md

```diff
@@ -244,7 +244,7 @@ Note: when signing a transaction, `txID`, `outputIndex`, `balanceRoot`, `stateRo
 
 Note: when verifying a predicate, `txID`, `outputIndex`, `balanceRoot`, `stateRoot`, and `txPointer` are initialized to zero.
 
-Note: when executing a script, `txID`, `outputIndex`, `balanceRoot`, and `stateRoot` are initialized to the transaction ID, output index, amount, and state root of the contract with ID `contractID`, and `txPointer` is initialized to zero.
+Note: when executing a script, `txID`, `outputIndex`, `balanceRoot`, `stateRoot` and `txPointer` are initialized to zero.
```

Comment:
Rather than initializing everything to zero, which is bad since we need the state and balance root set properly so they can be updated during execution for fraud proofs...you should instead set the `txPointer` properly! I.e. 804f6797a33e3727080c03b2d7c3d2ccf5dc92d3 was fine.